csharp/src/Ice/OutputStream.cs

```diff
@@ -413,12 +413,6 @@ public bool WriteOptional(int tag, OptionalFormat format)
         {
             Debug.Assert(_mainEncaps != null && _endpointEncaps == null);
 
-            if (Encoding.Equals(Util.Encoding_1_0))
-            {
-                // TODO: eliminate this block and return value
```

Comment:
Maybe we should keep a TODO to not forget about removing the return value.

Reply:
Added back new TODO.